Week1/homework/src/index.js

```diff
@@ -98,24 +90,24 @@
 
   let divRepDetails;
```

Comment:
You still have this global variable `divRepDetails` here. Try and get rid of it and pass a local variable `divRepDetails` as an argument to whatever function needs it.